src/Parser/VectorTileParser.js

```diff
@@ -4,12 +4,25 @@ import { VectorTile } from '@mapbox/vector-tile';
 import { globalExtentTMS } from 'Core/Geographic/Extent';
 import { FeatureCollection, FEATURE_TYPES } from 'Core/Feature';
 import { deprecatedParsingOptionsToNewOne } from 'Core/Deprecated/Undeprecator';
+import Coordinates from 'Core/Geographic/Coordinates';
 
 const worldDimension3857 = globalExtentTMS.get('EPSG:3857').planarDimensions();
 const globalExtent = new Vector3(worldDimension3857.x, worldDimension3857.y, 1);
 const lastPoint = new Vector2();
 const firstPoint = new Vector2();
 
+function project(x, y, tileNumbers, tileExtent) {
```

Comment:
Can you add some comments to avoid having to get into the maths to understand what is projected on what each time we come accross this function plase ?:)

Reply:
I added a lit comment to say where did i got the function from.